component/src/test/java/org/wso2/extension/siddhi/execution/streamingml/classification/perceptron/PerceptronClassifierStreamProcessorExtensionTestCase.java

```diff
@@ -116,7 +116,7 @@ public void receive(long timeStamp, Event[] inEvents, Event[] removeEvents) {
             AssertJUnit.assertEquals(4, count);
             Thread.sleep(1100);
         } catch (Exception e) {
-            logger.error(e.getMessage());
+            logger.error(e.getCause().getMessage());
```

Comment:
@niruhan shall we give the Exception as the next parameter value (eg: logger.error(e.getCause().getMessage(), e); check the other places as well.